modules/doc/content/newsletter/2020_03.md

```diff
@@ -1 +1,76 @@
 # MOOSE Newsletter (March 2020)
+
+## Stochastic Tools
+
+The [stochastic tools module](modules/stochastic_tools/index.md) is under heavy development and has
+gained some powerful features, with many more to come in the near future.
+
+1. Support for distributed VectorPostprocessor (VPP) data was added to the framework. This allows
+   massively parallel stochastic simulations to be performed with minimal memory impact. The VPP
+   must be programmed to support data that is distributed or replicated. The mode of operation
+   is controlled by the "parallel_type" parameter. When set to distributed mode an output file
+   for each processor will be created.
+
+2. The VPP statistics calculator was generalized to support replicated and distributed VPP data as
+   well as for multiple vectors, see [vectorpostprocessors/Statistics.md].
+
+3. Support for computing bootstrap confidence level intervals was added to the statistics calculator,
+   see [vectorpostprocessors/Statistics.md].
+
+4. An [XML](XMLOutput.md) output object was created to allow for single file VPP output.
+
+## Improved Mechanical and Thermal Contact Jacobians
+
+Jacobians for kinematic-formulation base mechanical contact were improved so that their accuracy is
+not impacted by variable scaling. This change results in significantly improved linear and nonlinear
+convergence for contact problems used in conjunction with scaling (either manual or
+automatic). Thermal contact Jacobians were also improved in the month of February. We now supply
+dependence of the thermal contact residuals on master-side degrees of freedom, including both
+(block) on-diagonal temperature and off-diagonal displacements degrees of freedom.
+
+## libMesh level changes
+
+### Adaptivity with LAGRANGE_VEC
+
+The libMesh update in the month of February enabled adaptivity with the LAGRANGE_VEC finite element
+family. The update also enables used of periodic boundary conditions with LAGRANGE_VEC.
+
+### Much faster periodic boundaries
+
+This was actually in a previous libMesh update, but the improved performance make it worth
+mentioniong here. Periodic boundaries now use a tree-search (O(log N) complexity) instead of a
```

Comment:
```suggestion
This was actually in a previous libMesh update, but the improved performance makes it worth
mentioning here. Periodic boundaries now use a tree-search (O(log N) complexity) instead of a
```